### ISSUE TITLE
webkit2gtk: cross build fails

### ISSUE THREAD

**Issue by pullmoll** [voidlinux/void-packages#5277]
Here we again have issues with `-isystem /usr/include`, similar to the `qt5` issue #5254.

This bug is being discussed here:
https://bugs.webkit.org/show_bug.cgi?id=161697

I currently have no workaround.

**Comment by ebfe** [comment]
http://trac.webkit.org/changeset/205672

**Comment by pullmoll** [comment]
I believe this patch is already included upstream.

**Comment by ebfe** [comment]
Yes, that's why non-cross builds work. Probably needs some adjustment to get cross builds working.

**Comment by ebfe** [comment]
Hmm.

non-cross
```
/builddir/webkitgtk-2.14.2/Source/cmake/OptionsCommon.cmake(59):  set(CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES /usr/include /usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/include;;/usr/local/include;;/usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/include-fixed;;/usr/include )
/builddir/webkitgtk-2.14.2/Source/cmake/OptionsCommon.cmake(64):  set(CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES /usr/include /usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/../../../../include/c++/6.2.1;;/usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/../../../../include/c++/6.2.1/x86_64-unknown-linux-gnu;;/usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/../../../../include/c++/6.2.1/backward;;/usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/include;;/usr/local/include;;/usr/lib/gcc/x86_64-unknown-linux-gnu/6.2.1/include-fixed;;/usr/include )
```

-a aarch64
```
/builddir/webkitgtk-2.14.2/Source/cmake/OptionsCommon.cmake(59):  set(CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES /usr/include /usr/lib/gcc/aarch64-linux-gnu/6.2.1/include;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/include-fixed;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/../../../../aarch64-linux-gnu/include )
/builddir/webkitgtk-2.14.2/Source/cmake/OptionsCommon.cmake(64):  set(CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES /usr/include /usr/lib/gcc/aarch64-linux-gnu/6.2.1/../../../../aarch64-linux-gnu/include/c++/6.2.1;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/../../../../aarch64-linux-gnu/include/c++/6.2.1/aarch64-linux-gnu;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/../../../../aarch64-linux-gnu/include/c++/6.2.1/backward;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/include;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/include-fixed;;/usr/lib/gcc/aarch64-linux-gnu/6.2.1/../../../../aarch64-linux-gnu/include )
```

**Comment by pullmoll** [comment]
I found a solution. See https://build.voidlinux.eu/changes/33406
